### ISSUE TITLE
[block-indentation] Embedded template after return statement has wrong columnOffset

### ISSUE THREAD

**Issue by robinborst95** [ember-template-lint/ember-template-lint#3164]
In our test suite, we have a few tests that do not do `await render`, but `return render`. For example:
```js
hooks.beforeEach(function() {
  this.renderTemplate = () => {
    return render(hbs`
      <div class="parent">
        <div class="child"></div>
      </div>
    `);
  };
});
```

This gives the impression that the template starts at column 2, while it actually starts at column 4. If I turn it into `await render`, it does work as expected.
